src/main/java/edu/ie3/util/geo/GeoUtils.java

```diff
@@ -211,6 +204,34 @@ public static ComparableQuantity<Length> calcHaversine(LineString lineString) {
     return y;
   }
 
+  /**
+   * Method to turn a distance into a latitude and longitude deltas. The methode can be found here:
+   * <a
+   * href="https://math.stackexchange.com/questions/474602/reverse-use-of-haversine-formula">source</a>
+   *
+   * @param coordinate the starting point for the calculation
+   * @param distance maximal distance in x- and y-direction
+   * @return x- and y-delta in degree as a coordinate
+   */
+  public static Coordinate calculateXYDelta(Point coordinate, ComparableQuantity<Length> distance) {
```

Comment:
I don't think it gets sufficiently clear what the purpose of this function is. Please elaborate what this method should do. What kind of delta do we calculate, what does it represent? It seems related to the haversine formula: What is the relation exactly? 

Also in the test only look at either the x or the y value of the coordinate you return: Why is that?